{{ cookiecutter.package_name }}/.coveragerc

```diff
@@ -0,0 +1,34 @@
+[run]
+omit = [
+ "{{ cookiecutter.module_name }}/_{{ cookiecutter._parent_project }}_init*",
+ "{{ cookiecutter.module_name }}/conftest.py",
+ "{{ cookiecutter.module_name }}/*setup_package*",
+ "{{ cookiecutter.module_name }}/tests/*",
+ "{{ cookiecutter.module_name }}/*/tests/*",
+ "{{ cookiecutter.module_name }}/extern/*",
+ "{{ cookiecutter.module_name }}/version*",
+ "*/{{ cookiecutter.module_name }}/_{{ cookiecutter._parent_project }}_init*",
+ "*/{{ cookiecutter.module_name }}/conftest.py",
+ "*/{{ cookiecutter.module_name }}/*setup_package*",
+ "*/{{ cookiecutter.module_name }}/tests/*",
+ "*/{{ cookiecutter.module_name }}/*/tests/*",
+ "*/{{ cookiecutter.module_name }}/extern/*",
+ "*/{{ cookiecutter.module_name }}/version*",
+]
+
+[report]
+exclude_lines = [
+  # Have to re-enable the standard pragma
+  "pragma: no cover",
+  # Don't complain about packages we have installed
+  "except ImportError",
+  # Don't complain if tests don't hit assertions
+  "raise AssertionError",
+  "raise NotImplementedError",
+  # Don't complain about script hooks
+  "def main(.*):",
+  # Ignore branches that don't pertain to this version of Python
+  "pragma: py{ignore_python_version}",
+  # Don't complain about IPython completion helper
+  "def _ipython_key_completions_"
```

Comment:
I think thr astropy list adds the overload one, we should get that added to the OA template?

Reply:
As we are moving this list out we are probably safe from conflicts to change it however we want here. That being said if there are things which are good improvements for upstream then we should do that too.

Reply:
I mean sure. The only difference between this and upstream is the overloads one. Outside of that I think its all the same.